injectable.gemspec

```diff
@@ -20,9 +20,9 @@ Gem::Specification.new do |spec|
   spec.bindir        = 'exe'
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
-  spec.required_ruby_version = '>= 2.5'
+  spec.required_ruby_version = '>= 2.7'
```

Comment:
🤔 I can't find that `.rubocop.yml` file inside of the repo, maybe it's something configured in a private repo that codeclimate picks up. I guess it's a good moment to bump that Ruby `2.1` up to `2.7` at least.

Reply:
I'm not sure where it's getting it from either. My guess, being such an old ruby version, is that it may be some kind of default value CodeClimate is setting, but I wasn't involved in setting that, we may have to take a look at it in the future